demisto_sdk/commands/common/tests/update_id_set_test.py

```diff
@@ -2776,6 +2782,187 @@ def test_process_jobs_file_nonexistent(capsys, repo, is_feed: bool, print_logs:
         assert f"failed to process job {job_json_path}" in capsys.readouterr().out
 
 
+class TestParsingRules:
+    @staticmethod
+    def test_process_parsing_rules(mocker, capsys, pack):
+        """
+        Given
+            - A repo with a parsing rule object.
+        When
+            - Parsing the parsing rule files.
+        Then
+            - Verify result as expeted.
+        """
+        mocker.patch.object(uis, 'should_skip_item_by_mp', return_value=False)
+        parsing_rule = pack.create_parsing_rule("parsing_rule", {"id": "parsing_rule_id", "rules": "", "name": "parsing_rule.yml"})
```

Comment:
why do we specify this: `"name": "parsing_rule.yml"`

Reply:
I did a small change in the tests
but I think it does not really matter what the name is